addons/l10n_in_edi/models/res_company.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    l10n_in_edi_username = fields.Char("E-invoice (IN) Username", groups="base.group_system")
+    l10n_in_edi_password = fields.Char("E-invoice (IN) Password", groups="base.group_system")
+    l10n_in_edi_token = fields.Char("E-invoice (IN) Token", groups="account.group_account_invoice")
```

Comment:
@jva-odoo why is this invoice and not system?

Reply:
@jco-odoo this is not what we show in the setting so invoice

Reply:
I am asking for this for security reasons: it is not something you want to share with a lot of people because it enables to invoice for a while on behalf of you.

Reply:
okay, let's move to the system group